### ISSUE TITLE
resource defined twice

### ISSUE THREAD

**Issue by seebi** [vocol/scor#8]
scor:ScheduleProductDeliveries

is once defined as

    rdfs:label "Schedule Product Deliveries"@en ;
    rdfs:subClassOf scor:SourceMakeToOrderProduct ;
    skos:notation "sS2.1" ;

and once as

    rdfs:label "Schedule Product Deliveries"@en ;
    rdfs:subClassOf scor:SourceEngineerToOrderProduct ;
    skos:notation "sS3.3" ;

pls re-identify one or both

**Comment by seebi** [comment]
three times ...

**Comment by seebi** [comment]
same for ReserveInventoryDetermineDeliveryDate, ConsolidateOrders, CrossDocking and DemandManagement

**Comment by np00** [comment]
Even though these concepts look very similar, they have tiny differences. Therefore, I introduced some small identifiers in the beginning of the name. Example:  

DSPReserveInventoryDetermineDeliveryDate

The first letter always reflects to the process category (D = Deliver) while the following ones give a hint on the subprocess class (SP = Sourced Product). 

ConsolidateOrders and DemandManagement refer to Skiils and Practises. Both are needed, thus I also added there a small identified (BP = Business Process, HS = "Human" Skill). 

See: https://github.com/vocol/scor/commit/3fb7ea84059bcc78c56204be9437655d3584fe69
